src/helpers/time.coffee

```diff
@@ -34,6 +34,11 @@ tzdetect =
 
 
 TimeHelper =
+
+  isPropAMoment: (props, propName, componentName) ->
```

Comment:
to match the naming of other `propTypes` could this be `TimeHelper.PropTypes.Moment`?